File: EncoraTestBLL/Models/Lease.cs
using System;
namespace EncoraTestBLL.Models
{
    public class Lease
    {

        #region class properties

        public int Id { get; set; }

        public LeaseSummary LeaseSummary { get; set; }

        public UtilitiesOwnership UtilitiesOwnership { get; set; }

        public ApplianceOwnership ApplianceOwnership { get; set; }

        #endregion

        public Lease()
        {
        }
    }
}

File: EncoraTest/ViewModels/lease.ts
import { LeaseSummary } from './leaseSummary'
import { UtilitiesOwnership } from './utilitiesOwnership'
import { ApplianceOwnership } from './applianceOwnership'

export interface Lease {
    leaseSummary: LeaseSummary;
    utilitiesOwnership: UtilitiesOwnership;
    applianceOwnership: ApplianceOwnership;
}